index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="responsive.css">
+    
+    <title>Document</title>
+</head>
+<body>
+  <div id="main">
+      <div id="title">
+          <h1>Nelson Mandela</h1>
+          <h5>The immortal leader of South Africa</h5>
+      </div>
+      <div id="img-div">
+          <img id="image" src="nelsonmandela.jpeg" alt="">
+          <p id="img-caption">The greatest photo taken of the greatest indeed!</p>
+      </div>
+      <div id="tribute-info">
+          <h3>The storyline of the G.O.A.T. Nelson Mandela</h3>
+          <ul>
+              <li>18 July 1918: Born in the village of Mvezo in Umtata, then part of South Africa's Cape Province.</li>
+              <li>1933: Began his secondary education at Clarkebury Methodist High School in Engcobo</li>
+              <li>1941: Fled to Johannesburg via Queenstown</li>
+              <li>1943: Mandela began studying law at the University of the Witwatersrand</li>
+              <li>1950: Mandela took Xuma's place on the ANC national executive</li>
+              <li>1955: Mandela concludes that violent action would prove necessary to end apartheid and white minority rule</li>
+              <li>1964: Mandela and his co-accused were transferred from Pretoria to the prison on Robben Island</li>
+              <li>2004: Mandela announced that he was "retiring from retirement" and retreating from public life</li>
+              <li>2011: Re-admitted for a lung infection and gallstone removal in December 2012.</li>
+              <li>2014: Announced dead</li>
+          </ul>
+          <div id="tribute-info">

```

Comment:
Please consider using classes over ids for multiple elements. Consider switching from id to class for the "tribute-info" since you can only use one id while you want to apply it to several.